servers-EXAMPLE.json

```
@@ -7,7 +7,7 @@
     },
     {
       "id": "bukkit",
-      "name": "Spoogot",
+      "name": "Spigot / Bukkit",
```

```suggestion
      "name": "Speegoot",
```

Bukkit's last releases were for 1.7.10 whereas this site doesn't track any earlier than 1.11, so I have suggested a fixed fix.